cla_backend/libs/eligibility_calculator/calculator.py

```diff
@@ -147,24 +145,22 @@ def _translate_section_gross_income(case_data, request_data):
         and put this in the CFE request.
         """
 
-        def is_gross_income_complete(case_data):
-            if not hasattr(case_data, "you"):
-                return False
-            person = case_data.you
-            if not hasattr(person, "income"):
-                return False
-            income = person.income
-
```

Comment:
I appreciate having this code... it explicitly deals with these cases, rather than figure out what evaluating a nested key will default to